site/app/templates/admin/admin_gradeable/AdminGradeableBase.twig

```diff
@@ -53,10 +55,10 @@
 <script>
 
     let adminGradeableNavTab = 0;
-    let navItemsStatus = new Map([['0',true],['1',true],['2',true],['3',true],['4',true]]);
+    var navItemsStatus = new Map([['0',true],['1',true],['2',true],['3',true],['4',true],['5',false]]);
```

Comment:
Is there a reason this being changed from 'var' to 'let', the scoping for 'var' works a bit weirder and can lead to some unexpected bugs because JS considered the scope to be the most immediate function instead of the local block you created that variable in so you might get a variable defined where you don't expect to be.

In this case it looks like it doesn't matter too much but we should try and use `let` and `const` because its a stronger signifier